test/templates/success.tpl.js

```diff
@@ -0,0 +1,32 @@
+// Loads the Kuzzle SDK module and the websocket protocol
```

Comment:
If I understand well the purpose of this template is to have code snippets like for the `default` template but without showing the `try { .. } catch { ... }` rights?
If so you can rename it accordingly